auth/customers.go

```diff
@@ -32,10 +32,8 @@ func buildCustomerURL() string {
 // IsCustomer returns true if a customer is listed in the customers file.
 // The validation is controlled by the 'validate_customers' env-var
 func IsCustomer(ownerLogin string, c *http.Client) (bool, error) {
-	validate := os.Getenv("validate_customers")
-
-	if len(validate) == 0 || (validate == "false" || validate == "0") {
-
+	validate := customerValidation()
```

Comment:
customerValidation could be `useCustomerValidation` or `customerValidationEnabled` to show the intent. It sounds a bike like an action as it is.

Reply:
Good idea, will quickly amend it